plugins/network-elements/nuage-vsp/pom.xml

```diff
@@ -28,6 +28,13 @@
     <version>4.6.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>${cs.lang3.version}</version>
+    </dependency>
+  </dependencies>
```

Comment:
Because our client has no dependencies to CloudStack, we had to find a common library between the 2 which has a Pair class. Which is <code>commons-lang3</code>